README.md

```diff
@@ -15,12 +15,15 @@
 This build should work for both macOS and Linux
 
 1. Download docker for your operating system
-2. From project root run the following commands:
-    * **Build:** `docker build -t point-service .`
-    * **Run:** `docker run -d=true -p 8083:8083 point-service`
+2. From project root run the following commands to build and run the app:
+    * `docker-compose up --build`
+    
+## Swagger
+
+Swagger can be accessed through http://localhost:8083/swagger-ui.html
```

Comment:
Love the swagger support. We have swagger on the recommendation microservice, but should definitely put it  in all of them